### PR TITLE
Implement copy-paste for rows in `TableBlock`

### PR DESCRIPTION
**Pull request by jamesricky**
## Related tasks and documents

https://vivid-planet.atlassian.net/browse/COM-1315

### CODE REVIEW

**Review by johnnyomair — COMMENTED** [review]
Should this really be done via the clipboard? Wouldn't in memory be sufficient?

File: packages/admin/cms-admin/src/blocks/table/ActionsCell.tsx [hunk 1]
```diff
@@ -74,6 +83,94 @@ export const ActionsCell = ({ row, updateState }: Props) => {
...
+
+        const copyData: ClipboardRow = {
+            highlighted: rowToCopy.highlighted,
+            cellValues: state.columns.map(({ id: columnId }) => {
+                const cellValue = rowToCopy.cellValues.find(({ columnId: cellColumnId }) => cellColumnId === columnId);
+                return cellValue ? cellValue.value : "";
+            }),
+        };
+
+        navigator.clipboard.writeText(JSON.stringify(copyData));
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/blocks/table/ActionsCell.tsx:107]: We already have functions that handle clipboard write/read with permission check and fallback here: https://github.com/vivid-planet/comet/tree/main/packages/admin/admin/src/clipboard

**Comment by jamesricky** [comment]
> Should this really be done via the clipboard?

I think it makes sense to handle this the same as copy/pasting blocks. 
This allows copying between different table-blocks, environments and projects. 

> Wouldn't in memory be sufficient?

What do you mean with "in memory"? 
Save it in a variable/state?

File: packages/admin/cms-admin/src/blocks/table/ActionsCell.tsx (hunk 1 above)
Comment (jamesricky) [reply packages/admin/cms-admin/src/blocks/table/ActionsCell.tsx]: Done: b5846d058a338b9fd88bdbce45431694dfd3ea98

File: packages/admin/cms-admin/src/blocks/table/ActionsCell.tsx [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
+import { z } from "zod";
 
 import { TableBlockData } from "../../blocks.generated";
-import { getNewRow } from "./utils";
+import { getNewColumn, getNewRow } from "./utils";
+
+const clipboardRowSchema = z.object({
+    highlighted: z.boolean(),
+    cellValues: z.array(z.string()),
+});
```
[the remark this answers is not in the document]
Comment (jamesricky) [reply packages/admin/cms-admin/src/blocks/table/ActionsCell.tsx:15]: Seemed like the easies way to validate if the structure of the pasted JSON is correct

[the remark this answers is not in the document]
Comment (jamesricky) [reply packages/admin/cms-admin/src/blocks/table/ActionsCell.tsx:15]: Do you know of a better alternative/method?

[the remark this answers is not in the document]
Comment (jamesricky) [reply packages/admin/cms-admin/src/blocks/table/ActionsCell.tsx:15]: @johnnyomair can we consider this resolved?

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**